recipes/aws-parallelcluster/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "aws-parallelcluster" %}
+{% set version = "2.3.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "d2759cc8724586ce75b566e1ef7d499740d4f8cf5d417cf52f24e47f5b6ca180"
+
+build:
+  number: 0
+  entry_points:
+    - pcluster = pcluster.cli:main
+    - awsbqueues = awsbatch.awsbqueues:main
+    - awsbhosts = awsbatch.awsbhosts:main
+    - awsbstat = awsbatch.awsbstat:main
+    - awsbkill = awsbatch.awsbkill:main
+    - awsbsub = awsbatch.awsbsub:main
+    - awsbout = awsbatch.awsbout:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - boto3 >=1.9.54
+    - future >=0.16.0,<=0.17.1
+    - pip
+    - python
+    - tabulate >=0.8.2,<=0.8.3
+    - setuptools
+    - configparser
+  run:
+    - boto3 >=1.9.54
+    - future >=0.16.0,<=0.17.1
```

Comment:
Does this version constraint need to be so restrictive?

Reply:
The version constraints are from the setup.py script of the package.  Since I'm not the package author, I don't feel comfortable relaxing the author-specified constraints.